src/locale/kbd-model-map

```diff
@@ -33,6 +33,7 @@ fr-pc			fr	pc105		-		terminate:ctrl_alt_bksp
 bg_pho-utf8		bg,us	pc105		,phonetic	terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
 it-ibm			it	pc105		-		terminate:ctrl_alt_bksp
 cz-us-qwertz		cz,us	pc105		-		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
+cz-qwerty		cz,us	pc105		qwerty		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
```

Comment:
Shouldn't there be `qwerty,` to match the number of layouts?

Reply:
oh, hmm. well that is what anaconda currently generates, but I thought it was a bug and sent a PR to 'fix' it: https://github.com/rhinstaller/anaconda/pull/4355 . However, you might be right that it was correct in the first place, I'll check your references and send a follow-up PR here if so. Thanks!

Reply:
Follow-up PR: https://github.com/systemd/systemd/pull/24868 . thanks again for catching this.